css/simplecal.css

```diff
@@ -48,7 +48,7 @@
     margin-top: 10px;
     background-color: var(--appblack);
     width: 100%;
-    text-align: right;
+    text-align: center;
```

Comment:
Can you please share the screenshots of what you have done?

Reply:
Hey, I haven't really tried this on my local system. I thought it'd correct it the way I wanted. 

Can you give me a few days to learn how to clone the repository in my system and see the changes? I haven't done this before.

Thank you.